### DISCUSSION TITLE
https://notes.alexkehayias.com/your-spam-folder-is-a-glimpse-of-what-censorship-free-content-looks-like/

### DISCUSSION THREAD

**Discussion by giscus** [alexkehayias/note-comments#8] [Announcements]
# https://notes.alexkehayias.com/your-spam-folder-is-a-glimpse-of-what-censorship-free-content-looks-like/

Centralized systems are criticized for making unilateral decisions about what people can see and not see within their privately controlled network.

https://notes.alexkehayias.com/your-spam-folder-is-a-glimpse-of-what-censorship-free-content-looks-like/

**Comment by jayceftg** [comment]
nah, bs gaslighting. spam gets quashed and truths get out in the censorship free public square. idts
